docs/src/main/asciidoc/azure-functions-http-guide.adoc

```diff
@@ -0,0 +1,120 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+
+include::./attributes.adoc[]
+= {project-name} - Using Resteasy, Undertow, or Vertx Web with Azure Functions (Serverless)
+
+The `quarkus-azure-functions-http` extension allows you to write microservices with Resteasy (JAX-RS),
```

Comment:
You of all people should be able to settle this: all the resteasy docs say it's `RESTEasy`, yet you use `Resteasy` here. How should we call it in Quarkus?

Reply:
I ended up never liking RESTEasy.  Looks like a typo.